frontend/src/components/PsyListing/publicPsychologistProfile.cssmodule.scss

```diff
@@ -1,13 +1,21 @@
+.psyName {
+  font-size: 24px;
+  font-weight: bold;
+  padding-bottom: 12px;
+}
+
 .field {
   padding: 12px 0px;
 }
 
 .fieldName {
   font-weight: bold;
+  padding-bottom: 5px;
 }
 
 .mapContainer {
-    width: 100%;
+  width: 100%;
+  margin-top: 12px;
```

Comment:
why do you need a margin top ?

Reply:
This margin-top is to add space between fields and map on mobile view:
![Selection_082](https://user-images.githubusercontent.com/68006779/125959932-95ed285f-5ee3-4cf7-87ad-1db4e77af689.jpg)

Reply:
then you can use the gap on display flex https://developer.mozilla.org/fr/docs/Web/CSS/gap